grafana-plugin/src/containers/ColumnsSelector/ColumnsSelector.styles.ts

```diff
@@ -0,0 +1,93 @@
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getColumnsSelectorStyles = (_theme: GrafanaTheme2) => {
```

Comment:
1) Can we use object literal syntax like css({ .... })? just as here: https://github.com/grafana/grafana/blob/main/contribute/style-guides/styling.md 
2) Shouldn't we use theme values for whitespaces (margins, paddings, gaps etc)?

Reply:
1. I feel like it's more closer to native css this way... we can discuss this further but it just feels more naturally writing it this way to me :thinking: 
2. I don't think we need to rely on these, the plan was to stick to emotion so that we fetch theme variables that are most important like colors, background colors etc and not hardcode them in css files. Margins don't scale all the time, could be 4, could be 6, could be a multiple of whatever X. For example there could be cases where you just need to adjust your margins because grafana appends too much on some elements, that margin may very well not be a multiple of a standard X value

Reply:
So regarding point 2) - as we are not using theme vars now, let's leave it. We will get back to this topic after full migration to emotion.

Regarding 1st point, I understand that at first it might seem more naturally to write it with string syntax because we're using plain css in the project right now but I believe after the switch it's only the matter of time to get use to writing styles in js/ts way. Two reasons why I would suggest the latter are:
1) Intellisense + type safety
![image](https://github.com/grafana/oncall/assets/12073649/99e7cec7-da94-4765-a508-e8aad75d3744)
2) At the end we're a part of Grafana and such style guides are created for a reason. If it is explicitly recommended to use a functional way and unless we have a solid argument to not follow it, why not to do it.

@maskin25 maybe you can share your thoughts?

Btw. it doesn't matter that much for this PR what we decide but we need to have this decision before migrating the whole codebase

Reply:
Also first point in best practices from Emotion docs: https://emotion.sh/docs/best-practices#use-typescript-and-object-styles

Reply:
1. object literal syntax looks more convenient, in general we should use it to make Intellisense and ts checking work
2. I think we need to arrange a session with @raphael-batte to discuss margins/paddings unification to be consistent with Grafana